index.html

```diff
@@ -0,0 +1,350 @@
+          <figcaption>
+            <h4 class="portfolio-caption-header">Flight control</h4>
+            <p class="portfolio-category">Webdesign, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+
+        <figure class="portfolio-item">
+          <a class="portfolio-link-hover" href=""><img class="portfolio-opacity" src="/img/ytube-app.jpg" alt="Image of Youtube app on a handheld device" title="Youtube app image"></a>
+          <figcaption>
+            <h4 class="portfolio-caption-header">Youtube app</h4>
+            <p class="portfolio-category">Icons design, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+
+        <figure class="portfolio-item">
+          <a class="portfolio-link-hover" href=""><img class="portfolio-opacity" src="/img/management.jpg" alt="Image of app screen for claim management" title="Claim management screen"></a>
+          <figcaption>
+            <h4 class="portfolio-caption-header">Management</h4>
+            <p class="portfolio-category">UI design, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+
+        <figure class="portfolio-item">
+          <a class="portfolio-link-hover" href=""><img class="portfolio-opacity" src="/img/korean.jpg" alt="Image of recipe app screen for korean bulgogi beef" title="Korean beef recipe screen"></a>
+          <figcaption>
+            <h4 class="portfolio-caption-header">Korean beef</h4>
+            <p class="portfolio-category">Webdesign, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+
+        <figure class="portfolio-item">
+          <a class="portfolio-link-hover" href=""><img class="portfolio-opacity" src="/img/facebook-thumb.jpg" alt="Image of Facebook like and dislike thumb icons" title="Facebook thumbs"></a>
+          <figcaption>
+            <h4 class="portfolio-caption-header">Facebook thumbs</h4>
+            <p class="portfolio-category">Icons design, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+
+      </article>
+      <article id="about" class="about">
+        <h2 class="about-header white-line">ABOUT</h2>
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
+        <section>
+          <h3 class="skills-header white-line">SKILLS</h3>
+          <ul class="skills">
+            <li>
+              <h4 class="progress-header">PHOTOSHOP</h4>
+              <progress value="77" max="100"></progress>
+            </li>
```

Comment:
Good use of `progress`, but I think the `progress` element and it's associated `h4` should be wrapped in a `figure`/`figcaption` structure.